extensions/ql-vscode/src/common/vscode/commands.ts

```diff
@@ -62,6 +64,16 @@ export function registerCommandWithErrorHandling(
         } else {
           void showAndLogWarningMessage(logger, errorMessage.fullMessage);
         }
+      } else if (e instanceof CliError) {
+        const fullMessage = `${e.commandDescription} failed with args:${EOL}${
```

Comment:
If I'm not mistaken, this message will come out as:
```
SomeCommand failed with args:
stderr stderr stderr stderr stderr stderr stderr stderr
stderr stderr stderr stderr stderr stderr stderr stderr
stderr stderr stderr stderr stderr stderr stderr stderr
stderr stderr stderr stderr stderr stderr stderr stderr
stderr stderr stderr stderr stderr stderr stderr stderr
stderr stderr stderr stderr stderr stderr stderr stderr
stderr stderr stderr stderr stderr stderr stderr stderr
stderr stderr stderr stderr stderr stderr stderr stderr
 arg1 arg2 arg3
```
Should the args come before the stderr?

Reply:
Yes, you're right. I've updated this to be correct.